packages/client/src/request-login.test.ts

```diff
@@ -1,11 +1,108 @@
-import { getLoginUrlAndCodeVerifier } from './request-login';
+import { DEFAULT_SCOPE_STRING } from './constants';
+import { getLoginUrlWithCodeVerifierAndState } from './request-login';
 
-test('getLoginUrlAndCodeVerifier', async () => {
-  const { url } = await getLoginUrlAndCodeVerifier({
-    baseUrl: 'http://logto.dev/oidc/auth',
-    clientId: 'foo',
-    scope: 'openid offline_access',
-    redirectUri: 'http://localhost:3000',
+const BASE_URL = 'http://logto.dev/oidc/auth';
+const CLIENT_ID = 'foo';
+const REDIRECT_URI = 'http://localhost:3000';
+
+describe('getLoginUrlAndCodeVerifier', () => {
```

Comment:
should be `getLoginUrlAndCodeVerifierAndState`

Reply:
Fixed.